python/dgl/utils.py

```diff
@@ -9,9 +10,35 @@
 from . import backend as F
 from . import ndarray as nd
 
+
+def STR2DTYPE():
+    """Convert string to backend dtype"""
+    return {
+        'int32': F.int32,
+        'int64': F.int64
+    }
+
+
+def DTYPE2STR():
+    """Convert backend dtype to string"""
+    return {
+        F.int32: 'int32',
+        F.int64: 'int64'
+    }
+
+
+def STR2NPDTYPE():
+    """Convert string to np dtype"""
+    return {
+        'int32': np.int32,
+        'int64': np.int64
+    }
```

Comment:
Why not directly define them as:
```python
STR2NPDTYPE = {
  'int32' : np.int32,
  'int64' : np.int64
}
```
Besides, you probably could just use `getattr` instead of having these dictionaries.

Reply:
STR2DTYPE() uses backend `F` which causes cyclical import. I'll refactor this part